pkg/logutil/http_logger.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logutil
+
+import (
+	"net/http"
+
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+// HTTPLogger is a Logger that outputs logs to ResponseWriter.
+type HTTPLogger struct {
+	writer         http.ResponseWriter
+	logger         *zap.Logger
+	closeCallbacks []func()
+}
+
+// NewHTTPLogger returns a HTTPLogger.
+func NewHTTPLogger(conf *log.Config, w http.ResponseWriter) (*HTTPLogger, error) {
+	syncer := zapcore.AddSync(w)
+	logger, props, err := log.InitLoggerWithWriteSyncer(conf, syncer, zap.AddStacktrace(zapcore.FatalLevel))
+	if err != nil {
+		return nil, err
+	}
+	props.Level.SetLevel(zapcore.DebugLevel)
```

Comment:
Why change it here?

Reply:
Specific debugging information requires more details.

Reply:
Why not `logger, props, err := log.InitLoggerWithWriteSyncer(conf, syncer, zap.AddStacktrace(zapcore. DebugLevel))`?

Reply:
I didn't know it :(